frontend/src/scenes/actions/ActionsTable.js

```diff
@@ -1,114 +1,105 @@
-                                            <div key={step.id}>
-                                                {(() => {
-                                                    switch (step.event) {
-                                                        case '$autocapture':
-                                                            return 'Autocapture'
-                                                        case '$pageview':
-                                                            return 'URL matches ' + step.url
-                                                        default:
-                                                            return 'Event: ' + step.event
-                                                    }
-                                                })()}
-                                            </div>
-                                        ))}
-                                    </td>
-                                    <td style={{ fontSize: 16 }}>
-                                        <Link to={'/action/' + action.id}>
-                                            <i className="fi flaticon-edit" />
-                                        </Link>
-                                        <DeleteWithUndo
-                                            endpoint="action"
-                                            object={action}
-                                            className="text-danger"
-                                            style={{ marginLeft: 8 }}
-                                            callback={this.fetchActions}
-                                        >
-                                            <i className="fi flaticon-basket" />
-                                        </DeleteWithUndo>
-                                    </td>
-                                </tr>
-                            ))}
-                    </tbody>
-                </table>
-            </div>
-        )
-    }
+    return (
+        <div>
+            <h1>Actions</h1>
+            <p style={{ maxWidth: 600 }}>
+                <i>
+                    Actions are PostHog’s way of easily cleaning up a large amount of Event data. Actions consist of one
+                    or more events that you have decided to put into a manually-labelled bucket. They're used in
+                    Funnels, Live actions and Trends.
+                    <br />
+                    <br />
+                    <a href="https://github.com/PostHog/posthog/wiki/Actions" target="_blank" rel="noopener noreferrer">
+                        See documentation
+                    </a>
+                </i>
+            </p>
+            <LinkButton to={'/action'} type="primary">
+                + New Action
+            </LinkButton>
+            <br />
+            <br />
+            <Table
+                size="small"
+                columns={columns}
+                loading={!actions && actionsLoading}
```

Comment:
`!actions` will always be true because it's default is set to `[]`. I think just passing `actionsLoading` here will be fine?

Reply:
Yes! Might be worth changing the funnels to do the same so logic is consistent